Tests/Unit/Microsoft365DSC/Microsoft365DSC.AADApplication.Tests.ps1

```diff
@@ -1,5 +1,5 @@
 [CmdletBinding()]
-param(
+                                                param(
```

Comment:
Please remove the added spaces, so the param keyword remains at the beginning of the line.